src/mappings/core.ts

```diff
@@ -26,12 +16,18 @@ import {
   createLiquidityPosition,
   ZERO_BD,
   BI_18,
-  createLiquiditySnapshot
+  createLiquiditySnapshot,
 } from './helpers'
-import { getFactoryAddress } from '../commons/addresses'
+import { getBundle, getSwaprFactory } from './factory'
 
 function isCompleteMint(mintId: string): boolean {
-  return MintEvent.load(mintId).sender !== null // sufficient checks
+  let mintEvent = MintEvent.load(mintId)
+
+  if (mintEvent && mintEvent.sender !== null) {
```

Comment:
What about `return mintEvent && mintEvent.sender !== null`?

Reply:
Addressed in [6051be0](https://github.com/dxgraphs/swapr-subgraph/pull/17/commits/6051be006ad9c9fab6a6705e92ae5433a0e92a2e)